dashboard/app/helpers/levels_helper.rb

```diff
@@ -1055,6 +1052,7 @@ def get_programming_environment
     end
   end
 
+  # This is what we need to modify so that it returns either one text string OR a blob
```

Comment:
super nit picky - this reads more like a TODO. Maybe reword the comment to just say that some labs support more than one file

Reply:
Agreed - somehow the changes I had made to `levels_helper` never got pushed (the "somehow" was probably "I forgot to push them before requesting review") - sorry about letting those puts statements and this "kaitie thinking out loud in a PR" sneak through.  I changed this comment